### ISSUE TITLE
Webviewer clears when fmGo is in background.

### ISSUE THREAD

**Issue by jwillinghalpern** [geistinteractive/fm-webviewer-bridge#2]
If I load a webviewer with the load script and then lock my iphone or navigate to another app, when I reopen fmGo the app I loaded is no longer in the webviewer. I can get "fix" this by saving the url with initial props into a $$global or field and set the wv calculation to that $$global.

If the wv content won't "stick" when fmGo is sent to the background, then maybe load wv script should return the compiled url in the result so that it can be easily saved? The alternative is to use getLayoutObjectAttribute ( "webviewer" ; "source" ) to query the url, but it's possible the url will have changed by then if the webapp changes its own hash value.

You might have a better idea for handling the disappearing wv contents though...

**Comment by toddgeist** [comment]
yeah, this is an issue with Web Viewers in FileMaker Go.  I think you have approximately the right solution.

**Comment by jwillinghalpern** [comment]
Thinking of a non-redundant way to manage that that preserves the current API. One way is: 
1. `Load Webviewer App` accepts a new optional param, a variable/field name target to store the url. It  saves the compiled url there.
2. If that param is provided, instead of doing _Set Web Viewer [ Object Name: $webViewer ; URL: $url ]_, it will assume the webviewer is pointed to that url and just refresh the webviewer. Or I suppose it could also just set the web viewer directly using the $$TARGET_URL with the same outcome, and without making config assumptions.

**Comment by toddgeist** [comment]
We have some new ideas that I think might address this.  Basically separating the compile step from the loading into the web viewer step.  I think this will address this.
